bin/logstash.bat

```diff
@@ -51,12 +54,38 @@ for %%i in ("%LS_HOME%\logstash-core\lib\jars\*.jar") do (
 
 %JAVA% %JAVA_OPTS% -cp %CLASSPATH% org.logstash.Logstash %*
 
-endlocal
+goto :end
+
+:version
+set "LOGSTASH_VERSION_FILE1=%LS_HOME%\logstash-core\versions-gem-copy.yml"
+set "LOGSTASH_VERSION_FILE2=%LS_HOME%\versions.yml"
+
+set "LOGSTASH_VERSION=Version not detected"
+if exist !LOGSTASH_VERSION_FILE1! (
+	rem this file is present in zip, deb and rpm artifacts and after bundle install
+	rem but might not be for a git checkout type install
+	for /F "tokens=1,2 delims=: " %%a in (!LOGSTASH_VERSION_FILE1!) do (
+		if "%%a"=="logstash" set LOGSTASH_VERSION=%%b
+	)
+) else (
+	if exist !LOGSTASH_VERSION_FILE2! (
+		rem this file is present for a git checkout type install
+		rem but its not in zip, deb and rpm artifacts (and in integration tests)
+		for /F "tokens=1,2 delims=: " %%a in (!LOGSTASH_VERSION_FILE2!) do (
+			if "%%a"=="logstash" set LOGSTASH_VERSION=%%b
+		)
+	)
+)
+echo "logstash !LOGSTASH_VERSION!"
```

Comment:
this outputs the following:
```
C:\Users\colin\dev\logstash (pr/8508)
λ bin\logstash -V
"logstash 7.0.0-alpha1"

C:\Users\colin\dev\logstash (pr/8508)
λ bin\logstash --version
"logstash 7.0.0-alpha1"
```

To be consistent with the Linux behaviour we should remote the quotes from the output, it should be 
```
echo logstash !LOGSTASH_VERSION!
```